testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -730,7 +742,25 @@ impl BitcoinRegtestController {
     ) -> Option<Transaction> {
         let public_key = signer.get_public_key();
 
-        let (mut tx, utxos) = self.prepare_tx(&public_key, payload.burn_fee, attempt)?;
+        let sunset_fee = if payload.sunset_burn > 0 {
+            cmp::max(payload.sunset_burn, DUST_UTXO_LIMIT)
+        } else {
+            0
+        };
+
+        let number_of_transfers = payload.commit_outs.len() as u64;
+        let value_per_transfer = payload.burn_fee / number_of_transfers;
```

Comment:
Does `value_per_transfer` need to be `= cmp::max(value_per_transfer, DUST_UTXO_LIMIT)`?

Reply:
I think in that case, you would go over the cap that was fixed by the miner. I think this is being checked at a higher level, but we can abort the transaction here as well.

Reply:
Ah, okay, as long as it gets checked somewhere, that works for me.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2317/commits/c654e94be572e5edabf9a3205f1fe5cbb336a8d6